src/NzbDrone.Core/Download/Clients/Porla/Porla.cs

```diff
@@ -0,0 +1,281 @@
+                    Title = torrent.Name,
+                    TotalSize = torrent.Size,
+                    SeedRatio = torrent.Ratio
+                };
+
+                // deal with moving_storage=true ?
+
+                if (!string.IsNullOrEmpty(torrent.Error))
+                {
+                    item.Status = DownloadItemStatus.Warning;
+                    item.Message = torrent.Error;
+                } // TODO: check paused or finished first?
+                else if (torrent.FinishedDuration > 0)
+                {
+                    item.Status = DownloadItemStatus.Completed;
+                    if (torrent.ETA < 1)
+                    {
+                        // Sonarr wants to see a TimeSpan.Zero when it is done ( -1 -> 0 )
+                        item.RemainingTime = TimeSpan.Zero;
+                    }
+                }
+                else if (torrent.Flags.Contains("paused"))
+                {
+                   item.Status = DownloadItemStatus.Paused;
+                } /*  I don't know what the torent looks like if it is "Queued"
+                else if (???)
+                {
+                   item.Status = DownloadItemStatus.Queued;
+                } */
+                else
+                {
+                    item.Status = DownloadItemStatus.Downloading;
+                }
+
+                item.CanMoveFiles = item.CanBeRemoved = true; // usure of what restricts this on porla. Currently these is always true
+
+                items.Add(item);
+            }
+
+            if (items.Count < 1)
+            {
+                _logger.Debug("No Items Returned");
+            }
+
+            return items;
+        }
+
+        public override void RemoveItem(DownloadClientItem item, bool deleteData)
+        {
+            // Kinda sucks we don't have a `RemoveItems`, porla has a batch interface for removals
+            PorlaTorrent[] singleItem = { new PorlaTorrent(item.DownloadId, "") };
+            _proxy.RemoveTorrent(Settings, deleteData, singleItem);
+
+            // when do we set item.Removed ?
+        }
+
+        public override DownloadClientInfo GetStatus()
+        {
+            var presetEffectiveSettings = _proxy.ListPresets(Settings).GetEffective(Settings.Preset.IsNullOrWhiteSpace() ? "default" : Settings.Preset);
+
+            // var sessionSettings = _proxy.GetSessionSettings(Settings);
+
+            var status = new DownloadClientInfo
+            {
+                IsLocalhost = Settings.Host == "127.0.0.1" || Settings.Host == "localhost",
+                RemovesCompletedDownloads = false, // TODO: I don't think porla has config for this, it feels like it should
+            };
+
+            var savePath = ((presetEffectiveSettings?.SavePath.IsNullOrWhiteSpace() ?? true) ? Settings.TvDirectory : presetEffectiveSettings?.SavePath) ?? "";
+            var destDir = new OsPath(savePath);
+            if (!destDir.IsEmpty)
```

Comment:
new line before blocks

```suggestion

            if (!destDir.IsEmpty)
```

Reply:
How did that slip in!
C# won't build of you screw up the formatting!